samtools.1

```diff
@@ -1263,6 +1263,24 @@ with the header in
 This command is much faster than replacing the header with a
 BAM\(->SAM\(->BAM conversion.
 
+By default this command outputs the BAM or CRAM file to standard
+output (stdout), but for CRAM format files it has the option to
+perform an insitu edit, both reading and writing to the same file.  No
+validity checking is performed on the header, nor that it is suitable
+to use with the sequence data itself.
+
+.B OPTIONS:
+.RS
+.TP 8
+.B -P, --no-PG
+Do not generate an @PG header line.
+.TP 8
+.B -I, --insitu
+Perform the header edit insitu, if possible.  This only works on CRAM
```

Comment:
`sed` and `perl` call this **in-place** editing, and use the options `-i[suffix]`/`--in-place[=suffix]`… but if you really really like _in situ_, I think it's two words :smile: